ContentManager.php

```diff
@@ -26,11 +27,11 @@ class ContentManager
     private array $handlers;
     private array $cache;
 
-    public function __construct(string $path, SerializerInterface $serializer, PropertyAccessorInterface $propertyAccessor)
+    public function __construct(string $path, SerializerInterface $serializer, PropertyAccessorInterface $propertyAccessor = null)
```

Comment:
```suggestion
    public function __construct(string $path, SerializerInterface $serializer, ?PropertyAccessorInterface $propertyAccessor = null)
```

(pur CS, mais on voit direct qu'un type est nullable via le typehint et non juste via default value)